man/systemd.network.xml

```diff
@@ -1617,6 +1617,13 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>HopLimit=</varname></term>
+        <listitem>
+          <para> Configures per route hop limit. See also <varname>IPv6HopLimit=</varname>.</para>
```

Comment:
Not a big matter, but unnecessary space after `<para>`.

Missing acceptable value range.